src/test/feature/gtest-parallel

```diff
@@ -234,6 +234,71 @@ class TestTimes(object):
     except IOError:
       pass  # ignore errors---saving the times isn't that important
 
+
+def get_schedule(schedule_file):
+  "read schedule file from --gtest_schedule option"
+  try:
+    file = open(schedule_file, 'r')
+  except (EOFError, IOError):
+    sys.exit("Read file error")
+  
+  filter_test = []
+  ptest = []
+  stest = []
+
+  for line in file.readlines():
```

Comment:
Actually I do not like the for-loop her because I think it is a little tricky. If you need a key-value dict information, could we change the file with json format?